packages/playground/website/src/github/github-export-form/modal.tsx

```diff
@@ -3,13 +3,19 @@ import { signal } from '@preact/signals-react';
 import Modal, { defaultStyles } from '../../components/modal';
 import GitHubExportForm, { GitHubExportFormProps } from './form';
 import { usePlaygroundClient } from '../../lib/use-playground-client';
+import { addURLState, removeURLState } from '../utils';
+import { PlaygroundDispatch } from '../../lib/state/redux/store';
+import { useDispatch } from 'react-redux';
+import { useEffect, useState } from 'react';
+import { setActiveModal } from '../../lib/state/redux/slice-ui';
 
 const query = new URLSearchParams(window.location.search);
 export const isGitHubExportModalOpen = signal(
 	query.get('state') === 'github-export'
```

Comment:
It seems like we could get rid of this signal and initialize the `activeModal` redux state using the `github-export` query parameter. For context – I introduced the signal way before we had the redux store and never got to remove it.